safe/gui/widgets/dock.py

```diff
@@ -930,6 +952,9 @@ def layer_changed(self, layer):
                         self.show_impact(table_report_path)
                         break
 
+            if self.impact_function.__class__ == MultiExposureImpactFunction:
```

Comment:
isinstance(self.impact_function, MultiExposureImpactFunction) is better I think than using using private members __XX__

Reply:
thanks